releases/point-2022-3.md

```diff
@@ -0,0 +1,81 @@
+        *The pre-defined metadata **set by the admin will be shown to the user as read-only**.
+    * For more information on **SysKit Point metadata, check out this article**.
+* **Service accounts can be added to an ignore list**.
+    * Setting up an ignored service account means:
+        * The Service Account page hits and views **will be ignored when detecting any workspace activity**.
+        * The Service Account audit logs **won’t be stored**.
+        * The Service Account changes **will be ignored and won’t trigger alerts**.
+    * Read more about **how to Ignore Service accounts here**.
+* **Added the ability to remove Sharing links** when generating the report.
+    * Select All feature available for removing all links attached to a certain Site.
+* **Configuration Inventory Module (CIM) settings have been added to SysKit Point**.
+    * Additional feature added that allows selecting who receives the CIM email.
+
+
+## Improvements & Bug Fixes
+
+* **Emails sent by SysKit Point now have customized dynamic subject field**.
+     * The subject line shows the context of the email; for example, Communication site is inactive.
+* **Improved the loading speed** for Access Review Options, **improved the functionality** of the preview email and dialogue on **the Access Review Options page**.
+* **Improved performance** for Privileged Identity Management (PIM).
+    * Privileged Identity Management (PIM) allows some users to assign themselves Global Admin rights for a limited amount of time.
+    * Global Admin sync improved to occurs every 4 minutes. 
+    * Fixed a bug where AAD groups weren’t taken into account for PMI if some members of those groups had administrative roles assigned.
+* **Improved speed and performance** for the One Drive tile when accessing through User details.
+* **Optimized speed and performance** when loading the Lifecycle Management details screen.
+* **Improved speed** for loading the Overview tile when accessing the Private Channel details screen.
+* **Improved messaging** when a Bad Request or Internal server error occurs that provides more insight into the issue that occurred when more information is available.
+* **Improved Access Review feature** to include exact dates when accessing the PR History Details Overview.
+  * Names of columns changed to include Status, Performed On and Performed By.
+    * Status bar states whether the action was Completed or Expired, and hovering above it provides information on who completed the task.
+* **Improved speed** for accessing the Lifecycle Management Email Preview.
+* **Fixed a bug** where the New Update Available ribbon was showing even after updating SysKit Point to the newest version.
+* **Fixed a bug** where the loading screen was cut off generating the details report.
+    * Once loading was completed, the report would be unresponsive when trying to select lines.
+* **Fixed an issue** where an error message was shown after opening the Details screen on Alerts.
```

Comment:
kod bugova napiši i o kojem se točno erroru radi, laše je tako onome tko čita RN vidjeti dal mu je ispravljen bug na koji je naletio ili ne
.. where a `Bad Request` error message was shown...